src/bsoncxx/include/bsoncxx/v1/document/value.hpp

```diff
@@ -66,6 +66,11 @@ class value {
     ///
     using default_deleter_type = std::default_delete<std::uint8_t[]>;
 
+    ///
+    /// The type of a pointer to @ref noop_deleter.
+    ///
+    using noop_deleter_type = void(BSONCXX_ABI_CDECL*)(std::uint8_t*);
```

Comment:
This is useful when inspecting the deleter, such as with `this->get_deleter().target<noop_deleter_type>()`.